.devcontainer/docker-compose-local.yaml

```diff
@@ -47,6 +49,7 @@ services:
           cpus: '2'
           memory: '5g'
     environment:
+      # ref???
```

Comment:
I'm not sure we need this

Reply:
remove for now, maybe let's get his PR shipped and iterate?